AugmentedNet/cli.py

```diff
@@ -39,15 +39,15 @@ class DefaultArguments(object):
         "outputRepresentations": [
             "Alto35",
             "Bass35",
-            # "ChordQuality11",
-            # "ChordRoot35",
+            "ChordQuality11",
+            "ChordRoot35",
             "HarmonicRhythm7",
-            # "Inversion4",
+            "Inversion4",
             "LocalKey38",
             "PitchClassSet121",
-            # "PrimaryDegree22",
+            "PrimaryDegree22",
             "RomanNumeral31",
-            # "SecondaryDegree22",
+            "SecondaryDegree22",
```

Comment:
Interesting. Are you running your experiments with all of these classification outputs turned on? I actually think I've never done that :).